packages/replayer/package.json

```diff
@@ -19,6 +19,7 @@
   },
   "dependencies": {
     "@alwaysmeticulous/common": "^2.22.0",
+    "@alwaysmeticulous/replay-debugger-ui": "^2.5.0",
```

Comment:
This makes standard replay depend on a lot more packages. Can we avoid it? I think it's fine to fail with `--debugger` if the package is not installed.

Reply:
We only use startServer from this package so I could peerDependency it and do a require if and only if running with '--debugger' flag:

```ts
  let startServer: StartServerFn;

  try {
    const replayDebugger = await require("@alwaysmeticulous/replay-debugger");
    startServer = replayDebugger.startServer;
  } catch (error) {
    logger.error("Error: could not import @alwaysmeticulous/replay-debugger-ui");
    logger.error(error);
    logger.error("");
    logger.error("Please make sure you've installed or added a dependency on '@alwaysmeticulous/replay-debugger-ui'. It is not installed automatically, and is required to use the '--debugger' flag.");
    process.exit(1);
  }
```

Reply:
Yes, you can lookup the `peerDependencies` in `cli` and see how to mark it as optional.